File: open_days_backend/src/main/java/com/sapientia/open/days/backend/ui/model/request/VerifyEmailByOtpCodeReq.java
package com.sapientia.open.days.backend.ui.model.request;

@SuppressWarnings("unused")
public class VerifyEmailByOtpCodeReq {
	private String email;

	private int otpCode;

	public String getEmail() {
		return email;
	}

	public int getOtpCode() {
		return otpCode;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public void setOtpCode(int otpCode) {
		this.otpCode = otpCode;
	}
}

File: open_days_frontend/lib/screens/registration/models/verify_email_by_otp_code_req.dart
class VerifyEmailByOtpCodeReq {
  int _otpCode;
  String _email;

  int getOtpCode() {
    return _otpCode;
  }

  String getEmail() {
    return _email;
  }

  void setOtpCode(int otpCode) {
    _otpCode = otpCode;
  }

  void setEmail(String email) {
    _email = email;
  }

  VerifyEmailByOtpCodeReq(this._email, this._otpCode);

  Map<String, dynamic> toJson() => {
        'email': _email,
        'otpCode': _otpCode,
      };
}
